src/components/pages/admin/events/updateSections/Details.js

```diff
@@ -328,13 +330,12 @@ class Details extends Component {
 				})
 				.catch(error => {
 					console.error(error);
-
 					notifications.showFromErrorResponse({
-						defaultMessage: "Loading venues failed",
+						defaultMessage: "Loading org/venue links failed.",
```

Comment:
The end user doesn't know or care about the difference between a venue and an org venue, let's also try make the error messages less "we've failed" and more "there was a problem, try again"
```suggestion
						defaultMessage: "There was a problem loading your venues",
```